packages/synapse-react-client/src/lib/containers/oauth/CreateOAuthClient.tsx

```diff
@@ -199,246 +214,224 @@ export const CreateOAuthModal: React.FunctionComponent<
     }
   }
 
-  return (
-    <div className="bootstrap-4-backport">
-      <Modal
-        show={isShowingModal && !isShowingConfirmModal}
-        animation={false}
-        backdrop="static"
-        onHide={() => {
-          hide()
-          setError(undefined)
+  const sectorIdentifierURIInput = (
```

Comment:
Consolidate Sector Identifier URI input definitions into one location for creating and editing a client